<launch>
<group ns="downward">
    <node pkg="nodelet" type="nodelet" name="camera_nodelet_manager"
          args="manager" />

    <!-- Left/Right Camera Driver Nodelets -->

    <!-- DOWNWARD CAMERA-->
    <node pkg="nodelet" type="nodelet" name="down_camera1394_nodelet"
          args="load camera1394/driver camera_nodelet_manager">
          <param name="guid" value="00b09d01009ce76f"/>
          <param name="frame_id" value="downward"/>
          <param name="num_dma_buffers" value="10"/>
          <param name="video_mode" value="format7_mode0"/>
          <param name="format7_packet_size" value="3008"/>
          <param name="format7_color_coding" value="raw8"/>
          <param name="binning_x" value="1"/>
          <param name="binning_y" value="1"/>
          <param name="auto_white_balance" value="2"/>
          <param name="use_ros_time" value="true"/>
          <remap from="camera" to="down"/>
    </node>

    <!-- mono rectification -->
    <node pkg="nodelet" type="nodelet" name="down_image_proc_rect"
          args="load image_proc/rectify camera_nodelet_manager">
      <remap from="image_mono" to="down/image_mono" />
      <remap from="image_rect" to="down/image_rect" />
    </node>

    <!-- color rectification -->
    <node pkg="nodelet" type="nodelet" name="down_image_proc_rect_color"
          args="load image_proc/rectify camera_nodelet_manager">
      <remap from="image_mono" to="down/image_color" />
      <remap from="image_rect" to="down/image_rect_color" />
    </node>

  </group>
</launch>
